leaflet.photon.js

```diff
@@ -31,8 +31,12 @@ L.PhotonBase = L.Class.extend({
     buildQueryString: function (params) {
         var queryString = [];
         for (var key in params) {
-            if (params[key]) {
-                queryString.push(encodeURIComponent(key) + '=' + encodeURIComponent(params[key]));
+            if (params[key] && (typeof(params[key]) == 'string' || typeof(params[key]) == 'number')) {
+                queryString.push(encodeURIComponent(key) + '=' + params[key]);
```

Comment:
This line should be factorized with the line 38.
Basically I think the pattern is to always process an array, and turn the param in an array when it's not.